docs/package_integration/uds-package-practices.md

```diff
@@ -1,110 +1,102 @@
+- **Must** implement Journey testing, covering the basic user flows and features of the application (see [Testing Guidelines](./testing-guidelines.md))
+  > This can by something like Playwright / [Cypress](https://github.com/defenseunicorns/uds-identity-config/tree/main/src/test/cypress) tests for services with a Web UI or something like [Jest](https://github.com/defenseunicorns/uds-package-gitlab-runner/tree/main/test) tests for headless services.
+- **Must** implement Upgrade Testing to ensure that the current development package works when deployed over the previously released one. (see [Testing Guidelines](./testing-guidelines.md))
+- **Must** be capable of operating within an internet-disconnected (air-gapped) environment
+- **Must** be actively maintained by the package maintainers identified in CODEOWNERS
+- **Must** be versioned using the UDS Package [Versioning scheme](#versioning)
+- **Must** contain documentation under a `docs` folder at the root that describes how to configure the package and outlines package dependencies.
+  > This allows users of the package to learn more about exposed configuration - it is recommended to make the entrypoint for configuration `configuration.md`.
+- **Must** have a dependency management bot (such as renovate) configured to open PRs to update core package and support dependencies.
+- **Must** release its package to the `ghcr.io/defenseunicorns/packages/<group>` namespace as the application's name (i.e. `ghcr.io/defenseunicorns/packages/uds/mattermost`).
+- **Must** not make the assumption that the `expose` interfaces are accessible to the bastion or pipeline deploying the package (i.e. `*.uds.dev`). If web requests need to be made they should be done through a `Job` or `./uds zarf tools kubectl exec` as appropriate.
 - **Should** lint their configurations with appropriate tooling such as [`yamllint`](https://github.com/adrienverge/yamllint) and [`zarf dev lint`](https://docs.zarf.dev/commands/zarf_dev_lint/).
 
-For more detailed guidelines on implementing these testing practices, please refer to the [Testing Guidelines for UDS Packages](./testing-guidelines.md).
-
-
-## Maintenance
+## Badging
 
-To help maintain a UDS Package, it:
-
-- **Must** have a dependency management bot (such as renovate) configured to open PRs to update core package and support dependencies.
+> [!NOTE]
+> The badge should link to the `uds-core` repo and should match the height of any other badges (i.e. `20px`).
 
-- **Must** release its package to the `ghcr.io/defenseunicorns/packages/<group>` namespace as the application's name (i.e. `ghcr.io/defenseunicorns/packages/uds/mattermost`).
+> [!NOTE]
+> Additional services will be added and integration requirements will change as `uds-core` continues to evolve. UDS Package implementers **should** keep up with `uds-core` releases and changes to this document. The "Made for UDS" badge **may** be removed if a package is not updated in a timely fashion.
```

Comment:
Do we want to solve this rather than have this statement? May be nice to lay out how this would go (i.e. Package is identified as not meeting standards, maintainer is notified, maintainer has 30 days to fix before their badge is downgraded or removed?)

Reply:
https://github.com/defenseunicorns/uds-common/issues/256 

I do want to solve this, but think we need to improve CI before we are able to do it effectively.